transform/lookups.go

```diff
@@ -92,3 +93,26 @@ func (c *company) municipio(l lookups, v string) error {
 	}
 	return nil
 }
+
+func (c *company) cnae(l lookups, v string) error {
+	cnae, err := getCnaeInfo(l, v)
+	if err != nil {
+		return err
+	}
+	c.CNAEFiscal = &cnae.Codigo
+	if cnae.Descricao != "" {
+		c.CNAEFiscalDescricao = &cnae.Descricao
+	}
+	return nil
+}
+func getCnaeInfo(l lookups, v string) (cnae, error) {
```

Comment:
Acho que essa função fica melhor perto de `type cnae struct` no `transform/company.go`, pois ela cria uma nova `cnae`.

Acho também que o nome pode ser melhorado: ela cria um `cnae`, enão eu tiraria o `Info` e usaria `new` ao invés de `get`:

```suggestion
func newCnae(l lookups, v string) (cnae, error) {
```